liveDeploys/index.html

```diff
@@ -27,16 +26,18 @@ <h5>{{liveDeploys.description}}</h5>
             <thead>
                 <tr>
                     <th>Name</th>
-                    <th>Profile Version</th>
+                    <th class="structured-data-column">Profile Version</th>
```

Comment:
Do not we need here the profile column as in the previous modified file (elixir.html)?

Reply:
No. The live deploys page is grouped by profile in the concertina table. On the elixir page they are grouped by node which is why we need the profile column as well.